src/vtpty/UnixUtils.cpp

```diff
@@ -3,25 +3,26 @@
 
 #include <cstring>
 #include <stdexcept>
-#include <string>
+
+#include <unistd.h>
 
 using namespace std::string_literals;
 
-namespace terminal
+namespace vtpty
 {
 
 UnixPipe::UnixPipe(unsigned flags): pfd { -1, -1 }
 {
 #if defined(__linux__)
-    if (pipe2(pfd, static_cast<int>(flags)) < 0)
+    if (pipe2(pfd.data(), static_cast<int>(flags)) < 0)
         throw std::runtime_error { "Failed to create PTY pipe. "s + strerror(errno) };
 #else
     if (pipe(pfd) < 0)
```

Comment:
what about this `pfd`? should it be also `.data()`?

Reply:
damn, nice catch.